inc/class_parser.php

```diff
@@ -1599,7 +1601,12 @@ function mycode_auto_url($message)
 		$message = my_substr($message, 1);
 
 		return $message;
-	}
+		}
+		else
+		{
+			return $message;
+		}
+		}
```

Comment:
Nitpick: this closing `}` should be outdented.

```suggestion
	}
```